docs/AssetsApi.md

```diff
@@ -138,7 +138,7 @@ end
 
 api_instance = MuxRuby::AssetsApi.new
 asset_id = 'asset_id_example' # String | The asset ID.
-create_track_request = MuxRuby::CreateTrackRequest.new # CreateTrackRequest | 
+create_track_request = {"url":"https://example.com/myVideo_en.srt","type":"text","text_type":"subtitles","language_code":"en-US","name":"English","closed_captions":true,"passthrough":"English"} # CreateTrackRequest | 
```

Comment:
I sent you a message about these things in Slack, I'd like to understand this more